docs/config.md

```diff
@@ -259,6 +259,11 @@ The max inherent score of a document in the final score.
 
     IMPORTANCE_WEIGHT = 0.1
 
+#### GEO_DISTANCE_WEIGHT (float)
+The max inherent score of geographical distance to the provided center (if any) in the final score.
+
+    GEO_DISTANCE_WEIGHT = 2.0
```

Comment:
We usually put here the default value :)

Reply:
Fixed :wink: